BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -12,6 +22,15 @@ def __init__(self, device: str, name: str, timestamp: str, status_code: str):
             self.t_red_start = timestamp
 
     def __create_event(self, timestamp):
+        """
+
+        To create the traffic light cycle event # Parameter: # argument1: timestamp (TimeStamp)
+        # Returns: # Device
```

Comment:
This does not return a device. This returns a `Tuple[str, str, str, str, str]` (device id, signal group, string representing the red light start time, string representing the green light start time, string representing the green light end time).